.github/workflows/check-pr-title.yml

```diff
@@ -3,6 +3,7 @@
 name: PR-Title
 
 on:
+  merge_group:
```

Comment:
does it work without any further information after?

Reply:
What do you mean? 

The only information I have is the official docs at https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue. I don't have any experience with how it will feel like.

Reply:
Ah, you mean this syntax:

```
on:
  merge_group:
  push:
```

This looks strange, indeed, but it's in the docs like this:
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions